src/components/ItemCard.jsx

```diff
@@ -6,22 +6,26 @@ import diy from '../images/defaultImages/diy.jpg'
 import kitchen from '../images/defaultImages/kitchen.jpg'
 import outdoors from '../images/defaultImages/outdoors.jpg'
 import tools from '../images/defaultImages/tools.jpg'
-import { borrowItemAsync, deleteItemAsync } from '../redux/items/thunks'
+import { borrowItemAsync, deleteItemAsync, updateItemAsync } from '../redux/items/thunks'
+import { ITEM_TYPES, STATUS } from '../constants'
 
 function ItemCard (props) {
   const borrowed = useSelector(state => {
-    return state.items.list.find((item) => item._id === props.id).status === 'Borrowed'
+    return state.items.list.find((item) => item._id === props.id).status === STATUS.BORROWED
```

Comment:
Could be wrong, but do you need to use selectors to get the item status? Wouldn't that be passed from the ItemContainer in `props`?

Reply:
sure, adjusting